phpunit.xml.dist

```diff
@@ -16,8 +16,10 @@
     convertNoticesToExceptions="true"
     convertWarningsToExceptions="true"
     enforceTimeLimit="true"
+    timeoutForSmallTests="2"
```

Comment:
To work around these failures: 
https://github.com/FriendsOfPHP/PHP-CS-Fixer/runs/1491007842#step:9:1010